dubbo-cluster/pom.xml

```diff
@@ -91,5 +92,10 @@
             <artifactId>micrometer-tracing-integration-test</artifactId>
             <scope>test</scope>
         </dependency>
+        <dependency>
+            <groupId>org.openjdk.nashorn</groupId>
+            <artifactId>nashorn-core</artifactId>
+            <version>${nashorn-core.version}</version>
+        </dependency>
```

Comment:
Test scope would be better. We should not import such engine by default.

Reply:
Okay, I'll change it to "testing".